handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslServerContext.java

```diff
@@ -153,6 +153,23 @@ static ServerContext newSessionContext(ReferenceCountedOpenSslContext thiz, long
                 } else {
                     SSLContext.setCertVerifyCallback(ctx, new TrustManagerVerifyCallback(engineMap, manager));
                 }
+
+                /* Disable to test if test timeouts are gone.
+                X509Certificate[] issuers = manager.getAcceptedIssuers();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)